ctrlf.el

```diff
@@ -99,7 +99,8 @@ lists with the following keys (all mandatory):
   '(([remap isearch-forward        ] . ctrlf-forward-literal)
     ([remap isearch-backward       ] . ctrlf-backward-literal)
     ([remap isearch-forward-regexp ] . ctrlf-forward-regexp)
-    ([remap isearch-backward-regexp] . ctrlf-backward-regexp))
+    ([remap isearch-backward-regexp] . ctrlf-backward-regexp)
+    ([remap isearch-forward-symbol-at-point] . ctrlf-forward-symbol-at-point))
```

Comment:
Do you want all of these realigned on the rightmost `.` now?

Reply:
Then nah cause cause it'll be a bigger diff and things will really slide to the right.